src/code/z_player_lib.c

```diff
@@ -1476,13 +1655,13 @@ s32 func_80091880(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* p
     return 0;
 }
 
-void func_80091A24(GlobalContext* globalCtx, void* seg04, void* seg06, SkelAnime* skelAnime, Vec3f* pos, Vec3s* rot,
-                   f32 scale, s32 sword, s32 tunic, s32 shield, s32 boots, s32 width, s32 height, Vec3f* eye, Vec3f* at,
-                   f32 fovy, void* img1, void* img2) {
+void Player_DrawPauseImpl(GlobalContext* globalCtx, void* seg04, void* seg06, SkelAnime* skelAnime, Vec3f* pos,
+                          Vec3s* rot, f32 scale, s32 sword, s32 tunic, s32 shield, s32 boots, s32 width, s32 height,
+                          Vec3f* eye, Vec3f* at, f32 fovy, void* colorFrameBuffer, void* depthFrameBuffer) {
     static Vp viewport = { 128, 224, 511, 0, 128, 224, 511, 0 };
```

Comment:
```suggestion
    static Vp viewport = { (64 / 2) << 2, (112 / 2) << 2, G_MAXZ / 2, 0, (64 / 2) << 2, (112 / 2) << 2, G_MAXZ / 2, 0 };
```
I think this is the correct way to do viewports. The image comes out to be 64x112 so exposing these explicitly seems more instructive (could make defines for the width and height if you feel like). I'm not totally sure if it's really worth to do `G_MAXZ / 2` rather than just leaving the value as-is though.